Doc/library/glob.rst

```diff
@@ -65,6 +65,7 @@ For example, ``'[?]'`` matches the character ``'?'``.
    match.
 
    .. audit-event:: glob.glob pathname,recursive glob.glob
+   .. audit-event:: glob.glob/2 pathname,root_dir,dir_fd,recursive glob.glob
```

Comment:
I don't see any particular reason to insert the new parameters compared to the previous event - why not keep the first two the same and just add the new ones on the end?

Reply:
I was just matching the order as presented in the function signature documentation. I've revised the PR to append new parameters at the end.